### PR TITLE
First Preview (v0)

### PR DESCRIPTION
**Pull request by 0exp**
- Code of Conduct (need a discussion here)
- Contributing rules (need a discussion here)
- Full example (in dev, need a discussion here)
- Workflow Design (need a discussion here)

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+### Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+### Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at [INSERT EMAIL ADDRESS]. All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (nesaulov) [inline CODE_OF_CONDUCT.md]: INSERT EMAIL ADDRESS

Comment (TrumpClone) [reply CODE_OF_CONDUCT.md]: @scatmanoleg ?

Comment (scatmanoleg) [reply CODE_OF_CONDUCT.md]: info@umbrellio.biz ?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,123 @@
 # UMBRELLIO guidelines
 
-In active development
+This manual describes the style and minimal requirements of maintaining and supporting the UMBRELLIO's github repositories.
+
+## Table of contents:
+
+* [Github Repository Design](#github-repository-design)
+  * Own projects
```
Comment (akxcv) [inline README.md]: Why is there no such paragraph, when there is for "Forks"?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,123 @@
...
+  * [Code of Conduct and Contributing](#code-of-conduct-and-contributing-rules)
+  * [Versioning](#versioning)
+  * [Changelog](#changelog)
+* [Full Example](#full-example)
+
+## Github Repositry Design
+
+### Title
+
+Minimal description of the repository. Should describe the main objective of the project. Clear and correct formuatlion is needed because it affects the SEO part.
```
Comment (akxcv) [inline README.md]: there is a typo in "formulation".

File: README.md [hunk 4]
```diff
@@ -1,3 +1,123 @@
...
+  - Minimal project requrements such as platform, operating system, native dependencies and etc.
+- Part 2. Installation
+  - Detailed instructions for installing the project and some specific notes.
+- Part 3. Usage
+  - The main documentation section. Arbitary form. Can be placed in the wiki section of the repository (or some of them with symlinks).
+
+### Links
+
+Links to arcticles that refers to the current project (articles of other companies, our articles, Medium links, reddit topics and etc).
+For the high popular projects this section is optional.
```
Comment (akxcv) [inline README.md]: For highly popular projects, you mean? 😕

Comment (0exp) [reply README.md]: yep

File: README.md (hunk 3 above)
Comment (0exp) [reply README.md]: yep

File: README.md (hunk 2 above)
Comment (0exp) [reply README.md]: Yep, useless section.

File: README.md [hunk 5]
```diff
@@ -1,3 +1,122 @@
...
+
+Provide a link to our [Code of Conduct](https://github.com/umbrellio/guidelines/blob/master/CODE_OF_CONDUCT.md)
+
+### License / Copyright / About Us
+
+Main sections:
+- License: short description, license type, link to the license file;
+- About us: briefly shows the company ideals and the list of the project maintainers and top contributors.
+  Basic form:
+  - (reuired) [company logo with sponsorship label](https://github.com/umbrellio/Umbrellio);
```
Comment (enthrops) [inline README.md]: required

**Review by enthrops — CHANGES_REQUESTED** [review]

**Review by enthrops — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -1,3 +1,122 @@
...
+
+Links to arcticles that refers to the current project (articles of other companies, our articles, Medium links, reddit topics and etc).
+For the highly popular projects this section is optional.
+
+### Contributing / Code of Conduct
+
+Provide a link to our [Contributing Rules](https://github.com/umbrellio/guidelines/blob/master/CONTRIBUTING.md)
+
+Provide a link to our [Code of Conduct](https://github.com/umbrellio/guidelines/blob/master/CODE_OF_CONDUCT.md)
+
```
Comment (enthrops) [inline README.md]: or put these files in the repo itself

**Review by enthrops — CHANGES_REQUESTED** [review]

**Review by enthrops — APPROVED** [review]

**Pull request merged**